test/jointjs/dia/Paper.js

```diff
@@ -95,6 +95,294 @@ QUnit.module('joint.dia.Paper', function(hooks) {
         });
     });
 
+    QUnit.module('transformToFitContent', function() {
+
+        hooks.beforeEach(function() {
+            const testGraph = new joint.dia.Graph();
+            testGraph.addCells([
```

Comment:
If I can change the `standard.Path` to `standard.Rectangle` and remove the `attrs` (`d` and `text`) and the test is still passing, please simplify the JSON. It only makes the test harder to read.